src/crab/ebpf_domain.hpp

```diff
@@ -382,6 +382,16 @@ class ebpf_domain_t final {
         for (const Instruction& statement : bb) {
             std::visit(*this, statement);
         }
+        if (global_options.check_termination) {
+            // +1 to avoid being tricked by empty loops
+            add(variable_t::instruction_count(), z_number((unsigned)bb.size() + 1));
+        }
+    }
+
+    bool terminates() {
+        using namespace crab::dsl_syntax;
+        constexpr int max_instructions = 100000;
```

Comment:
Maybe make the max_instructions default be overridable by a command line option?

Reply:
I don't see how it would be used. Prevail can even simply output a bound on the number of instructions and let the OS (or any other environment) decide whether or not it's too high.